<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(136, 165, 211);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="MainElements_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QWidget" name="profileWidget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="UserName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
border-radius: 40px;
</string>
             </property>
             <property name="text">
              <string>User</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="ExitButton">
             <property name="text">
              <string>Exit Profile</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="ChatsWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLineEdit" name="SearchUser"/>
           </item>
           <item>
            <widget class="QPushButton" name="SearchUserButton">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="UserList_2"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="ConversationWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="bottomMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="ChatName">
             <property name="text">
              <string>Chat Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="Messages"/>
           </item>
           <item>
            <widget class="QLineEdit" name="EnterMessage"/>
           </item>
           <item>
            <widget class="QPushButton" name="SendButton">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="ButtonLayout_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="ShowOrHideProfileButton">
           <property name="text">
            <string>Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
